src/interchange/src/avro.rs

```diff
@@ -146,10 +147,19 @@ pub struct DebeziumSourceCoordinates {
     snapshot: bool,
     row: RowCoordinates,
 }
+
+#[derive(Debug)]
+pub struct DebeziumTransactionMetadata {
+    // The order of the record within the transaction
+    total_order: usize,
```

Comment:
Transaction also contains `data_collection_order` and `id` -- are those fields useful?

Reply:
`id` is not really useful, for our purposes. `data_collection_order` is apparently per-table, as opposed to global for the transaction.